src/api/client.ts

```diff
@@ -7,39 +7,45 @@ import {
   searchCollectionsFieldPolicy,
 } from './field-policies';
 
-const apolloOptions = {
-  cache: new InMemoryCache({
-    typePolicies: {
-      CollectionAuthor: {
-        keyFields: ['externalId'],
-      },
-      CollectionStory: {
-        keyFields: ['externalId'],
-      },
-      Collection: {
-        keyFields: ['externalId'],
-      },
-      ApprovedCuratedCorpusItem: {
-        keyFields: ['externalId'],
-      },
-      RejectedCuratedCorpusItem: {
-        keyFields: ['externalId'],
-      },
-      ScheduledCuratedCorpusItem: {
-        keyFields: ['externalId'],
-      },
-      ScheduledCuratedCorpusItemsResult: {
-        keyFields: ['scheduledDate'],
-      },
-      Query: {
-        fields: {
-          getCollectionAuthors: getCollectionAuthorsFieldPolicy,
-          getCollectionPartners: getCollectionPartnersFieldPolicy,
-          searchCollections: searchCollectionsFieldPolicy,
-        },
+/**
+ * Custom type policies for Apollo Cache. Now in their own
+ * variable to feed them into integration tests, too.
+ */
+export const apolloCache = new InMemoryCache({
+  typePolicies: {
+    CollectionAuthor: {
+      keyFields: ['externalId'],
+    },
+    CollectionStory: {
+      keyFields: ['externalId'],
+    },
+    Collection: {
+      keyFields: ['externalId'],
+    },
+    ApprovedCorpusItem: {
+      keyFields: ['externalId'],
+    },
+    RejectedCorpusItem: {
+      keyFields: ['externalId'],
+    },
+    ScheduledCorpusItem: {
+      keyFields: ['externalId'],
+    },
```

Comment:
Along with the refactor into a variable, I updated the type names here - these were pre-refactor names with `Curated` in them and weren't actually having any effect, which may explain the issues with caching we had on the Corpus side of the app.